integration-tests/gradle/src/main/resources/implementation-files/application-files/build.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id 'io.quarkus'
+}
+
+dependencies {
+    implementation files("../common/build/libs/common.jar")
```

Comment:
Dependencies on files w/o "proper" dependencies was always a bad idea, and is strictly enforced w/ Gradle 8.
I left the cross-project task dependencies here, although cross-project tasks should also never be used (this is a test, so ...). The correct way here is to use a `project(":common")` dependency, which was added to this test as well.
Note: If "raw" files "from somewhere else" are really needed, then either use project+configuration (in the same build) or "published" artifacts generated by other projects.